content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -259,4 +264,7 @@ for `kube-reserved` and `system-reserved`.
 As of Kubernetes version 1.8, the `storage` key name was changed to `ephemeral-storage`
 for the alpha release.
 
+As of Kubernetes version 1.17, support for reserving hugepages via `hugepages-<size>`
```

Comment:
I recommend dropping this section and using a [feature-state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) instead. I would only show the v1.17 feature gate, the others are older than the oldest supported Kubernetes release.